ydb/core/base/table_vector_index.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+namespace NKikimr {
+namespace NTableIndex {
+namespace NTableVectorKmeansTreeIndex {
+
+// Vector KmeansTree index tables description
+
+// Levels table
+inline constexpr char LevelTable[] = "indexImplLevelTable";
+inline constexpr char LevelTable_ParentIdColumn[] = "-parent";
+inline constexpr char LevelTable_IdColumn[] = "-id";
+inline constexpr char LevelTable_EmbeddingColumn[] = "-embedding";
```

Comment:
Why do you need char array?
const char* or std::string_view should be enough

Reply:
Fixed